src/cargo/ops/cargo_update.rs

```diff
@@ -554,6 +777,92 @@ fn fill_with_deps<'a>(
     }
 }
 
+/// Upgrade an existing requirement to a new version.
+/// Copied from cargo-edit.
+fn upgrade_requirement(req: &str, version: &semver::Version) -> CargoResult<Option<String>> {
```

Comment:
nit: if we want to reduce the size of `cargo_update.rs`, this might also make sense to live in `toml_mut`.

Reply:
Fixed.